cockroachcloud/quickstart.md

```diff
@@ -44,40 +44,41 @@ If you have not done so already, install the CockroachDB binary:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    brew install cockroachdb/tap/cockroach
+    $ brew install cockroachdb/tap/cockroach
     ~~~
-    
+
 </section>
-    
+
 <section class="filter-content" markdown="1" data-scope="linux">
 
-1. In your terminal, run the following command to download and extract the CockroachDB {{ page.release_info.version }} archive for Linux:
+1. In your terminal, download the CockroachDB binary:
 
-    {% include_cached copy-clipboard.html %}
```

Comment:
@gemma-shay, heads up that this PR also fixes this few mistakes in the quickstart.